api/lib_test.go

```diff
@@ -197,12 +197,11 @@ func TestGetMetrics(t *testing.T) {
 
 	// GetMetrics 3
 	metrics, err = GetMetrics(cache)
-	require.NoError(t, err)
-	assert.Equal(t, &types.Metrics{
-		HitsFsCache:         1,
-		ElementsMemoryCache: 1,
-		SizeMemoryCache:     5665691,
-	}, metrics)
+	assert.NoError(t, err)
```

Comment:
Why do you replace `require` with `assert`?

Reply:
The same fixes as upstream have been made.

https://github.com/CosmWasm/wasmvm/blob/2857d72b920e2db3ba77592e56757e7ceb7693b7/internal/api/lib_test.go#L200

Reply:
It is ok if the aim is decreasing conflicts.